integration/key/key.go

```diff
@@ -40,7 +44,7 @@ func DefaultCatalogStorageURL() string {
 	return "https://giantswarm.github.io/default-catalog"
 }
 
-func Namespace() string {
+func GiantSwarmNamespace() string {
```

Comment:
This makes the diff a mess but I think its needed to make it clearer which namespace this is.